.github/workflows/rust.yml

```diff
@@ -80,7 +80,7 @@ jobs:
             uname -a
           run: |
             export PATH=$HOME/.rust_solaris/bin:$PATH
-            cargo build --all-targets && cargo test -- --ignored stress && cargo test --manifest-path fuzz/Cargo.toml && cargo test -p quinn-udp --benches
+            cargo build --all-targets && cargo test --manifest-path fuzz/Cargo.toml && cargo test -p quinn-udp --benches
```

Comment:
Maybe leave a comment explaining that ignoring the stress tests here is intended?